templates/header.hbs

```diff
@@ -1,27 +1,32 @@
 <a class="skip-navigation" tabindex="1" href="#main-content">{{t 'skip_navigation' }}</a>
 
 <header class="header">
-  <div class="logo">
-    {{#link 'help_center'}}
-      <img src="{{settings.logo}}" alt="{{t 'home_page' name=help_center.name}}" />
-      {{#if settings.show_brand_name}}
-        <span aria-hidden="true">{{help_center.name}}</span>
-      {{/if}}
+  <div class="header-left">
+    <div class="logo">
+      <a href={{settings.xpro_link}} class="xpro-link"> 
+        <img src="{{settings.logo}}" class="site-logo" />
+      </a>
+    </div>
+    {{#link 'help_center' class="support-center"}}
+      SUPPORT CENTER
     {{/link}}
+
   </div>
 
   <div class="nav-wrapper-desktop">
     <nav class="user-nav" id="user-nav" aria-label="{{t 'user_navigation'}}">
       <ul class="user-nav-list">
-        <li>{{link 'community'}}</li>
-        <li>{{link 'new_request' class='submit-a-request'}}</li>
-        {{#unless signed_in}}
-          <li>
-            {{#link "sign_in" class="sign-in"}}
-              {{t 'sign_in'}}
-            {{/link}}
-          </li>
-        {{/unless}}
+{{!--         <li>{{link 'community'}}</li> --}}
```

Comment:
Should we remove this commented code?

Reply:
I intentionally commented out this code, as well as other instances you may notice. The reason for this is to avoid large diffs and conflicts when resyncing with the upstream master in the future.

Reply:
Well in this case, it would be ideal to add an informative comment message saying why we are commenting it